subgrounds/pagination/strategies.py

```diff
@@ -485,8 +537,34 @@ def __init__(self, schema: SchemaMeta, document: Document) -> None:
         self.normalized_doc = normalize(schema, document, pagination_nodes)
 
     def step(
-        self, page_data: Optional[dict[str, Any]] = None
-    ) -> Tuple[Document, dict[str, Any]]:
+        self, page_data: dict[str, Any] | None = None
+    ) -> tuple[Document, dict[str, Any]]:
         args = self.arg_generator.step(page_data)
         trimmed_doc = prune_doc(self.normalized_doc, args)
         return (trimmed_doc, args)
+
+
+class SkipStrategy:
+    """This strategy always raises a :class:`~subgrounds.pagination.SkipPagination`
+      when constructed or if *somehow* `step` is called.
+
+    This is the default if `None` is passed to the strategy (default for non-subgraphs).
+    """
+
+    def __init__(self, schema: SchemaMeta, document: Document) -> None:
+        raise SkipPagination
+
+    def step(
+        self, page_data: Optional[dict[str, Any]] = None
+    ) -> tuple[Document, dict[str, Any]]:
+        raise SkipPagination
```

Comment:
`step` should probably never be called since you can't construct this object. It should likely crash explictly, but perhaps this is the wrong `Exception` to use for this 🤔.